packages/cli/templates/actions/empty-browser-action/__tests__/index.test.ts

```diff
@@ -0,0 +1,9 @@
+import { Analytics, Context } from '@segment/analytics-next'
+import * as jsdom from 'jsdom'
+import { destination } from '../../index'
+
+describe('{{destination}}', () => {
+  describe('{{name}}', () => {
```

Comment:
I think it's best practice to not have multiple of the same `describe` names, which would happen here if you have more than one action for a destination.

What about simply:
```suggestion
describe('{{destination}}.{{slug}}', () => {
```